main.tf

```diff
@@ -156,10 +157,11 @@ resource "aws_ecs_task_definition" "task" {
     "secrets": ${jsonencode(var.task_container_secrets)},
     %{~endif}
     "essential": true,
+    "privileged": ${var.privileged ? "true" : "false"},
```

Comment:
This should be only `var.privileged` bare as it will be converted to string by Terraform.

Reply:
You're right